rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1255,10 +1251,12 @@ def _create_sequence(
 
     def _create_all_labels(self) -> Tuple[tf.Tensor, tf.Tensor]:
         all_label_ids = self.tf_label_data[LABEL_IDS][0]
+
+        label_lengths = tf.cast(self.tf_label_data[LABEL_SEQ_LENGTH][0], dtype=tf.int32)
```

Comment:
Should we add a helper method for this? Something like `sequence_length_for`? This logic is repeated all over the place.